pkg/route/api/helper.go

```diff
@@ -20,11 +20,16 @@ func RouteLessThan(route1, route2 *Route) bool {
 	if route1.CreationTimestamp.Before(route2.CreationTimestamp) {
```

Comment:
Please change the commit description to be more descriptive of the change :)  It massively helps me when I'm trying to draft release notes and need to read through 1000+ commits if the subject of each commit reflects the actual change.